<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameBoard</class>
 <widget class="QMainWindow" name="GameBoard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1129</width>
    <height>809</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TIC TAC TOE</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="button_0_2">
    <property name="geometry">
     <rect>
      <x>889</x>
      <y>100</y>
      <width>181</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>The Sunday</family>
      <pointsize>72</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 72pt &quot;The Sunday&quot;;
    background-color: transparent;
    border: none;
    color: #FFFFFF; /* Text color */
}

QPushButton:hover {
    background-color: transparent;
    border: none;
    color: #CCCCCC; /* Optional: change text color slightly on hover */
}

QPushButton:pressed {
    background-color: transparent;
    border: none;
    color: #AAAAAA; /* Optional: darker text when pressed */
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_0_1">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>110</y>
      <width>181</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>The Sunday</family>
      <pointsize>72</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">

QPushButton {
	font: 72pt &quot;The Sunday&quot;;
    background-color: transparent;
    border: none;
    color: #FFFFFF; /* Text color */
}

QPushButton:hover {
    background-color: transparent;
    border: none;
    color: #CCCCCC; /* Optional: change text color slightly on hover */
}

QPushButton:pressed {
    background-color: transparent;
    border: none;
    color: #AAAAAA; /* Optional: darker text when pressed */
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_2_0">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>529</y>
      <width>191</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>The Sunday</family>
      <pointsize>72</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 72pt &quot;The Sunday&quot;;
    background-color: transparent;
    border: none;
    color: #FFFFFF; /* Text color */
}

QPushButton:hover {
    background-color: transparent;
    border: none;
    color: #CCCCCC; /* Optional: change text color slightly on hover */
}

QPushButton:pressed {
    background-color: transparent;
    border: none;
    color: #AAAAAA; /* Optional: darker text when pressed */
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="scoreLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>371</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>The Sunday</family>
      <pointsize>14</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;The Sunday&quot;;
    color: #030554;
    border: 2px solid #01004c;
    border-radius: 10px;
    padding: 5px;
    background-color: white;</string>
    </property>
    <property name="text">
     <string>SCORE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ReturnButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>570</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>The Sunday</family>
      <pointsize>14</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton 
{ 
background-color:  #030554;
      color: white;
        font-size: 20px;
	font: 14pt &quot;The Sunday&quot;;
        border-radius: 30px;
        padding: 10px 20px
}</string>
    </property>
    <property name="text">
     <string>Return Mode Window</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_1_0">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>310</y>
      <width>191</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>The Sunday</family>
      <pointsize>72</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 72pt &quot;The Sunday&quot;;
    background-color: transparent;
    border: none;
    color: #FFFFFF; /* Text color */
}

QPushButton:hover {
    background-color: transparent;
    border: none;
    color: #CCCCCC; /* Optional: change text color slightly on hover */
}

QPushButton:pressed {
    background-color: transparent;
    border: none;
    color: #AAAAAA; /* Optional: darker text when pressed */
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_2_2">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>530</y>
      <width>181</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>The Sunday</family>
      <pointsize>72</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 72pt &quot;The Sunday&quot;;
    background-color: transparent;
    border: none;
    color: #FFFFFF; /* Text color */
}

QPushButton:hover {
    background-color: transparent;
    border: none;
    color: #CCCCCC; /* Optional: change text color slightly on hover */
}

QPushButton:pressed {
    background-color: transparent;
    border: none;
    color: #AAAAAA; /* Optional: darker text when pressed */
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_0_0">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>100</y>
      <width>191</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>The Sunday</family>
      <pointsize>72</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">

QPushButton {
	font: 72pt &quot;The Sunday&quot;;
    background-color: transparent;
    border: none;
    color: #FFFFFF; /* Text color */
}

QPushButton:hover {
    background-color: transparent;
    border: none;
    color: #CCCCCC; /* Optional: change text color slightly on hover */
}

QPushButton:pressed {
    background-color: transparent;
    border: none;
    color: #AAAAAA; /* Optional: darker text when pressed */
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>650</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>The Sunday</family>
      <pointsize>14</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton 
{ 
background-color:  #030554;
      color: white;
        font-size: 20px;
	font: 14pt &quot;The Sunday&quot;;
        border-radius: 30px;
        padding: 10px 20px
}</string>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>371</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>The Sunday</family>
      <pointsize>14</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;The Sunday&quot;;
    color: #030554;
    border: 2px solid #01004c;
    border-radius: 10px;
    padding: 5px;
    background-color: white;</string>
    </property>
    <property name="text">
     <string>STATUS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_1_1">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>310</y>
      <width>181</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>The Sunday</family>
      <pointsize>72</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 72pt &quot;The Sunday&quot;;
    background-color: transparent;
    border: none;
    color: #FFFFFF; /* Text color */
}

QPushButton:hover {
    background-color: transparent;
    border: none;
    color: #CCCCCC; /* Optional: change text color slightly on hover */
}

QPushButton:pressed {
    background-color: transparent;
    border: none;
    color: #AAAAAA; /* Optional: darker text when pressed */
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_2_1">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>530</y>
      <width>181</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>The Sunday</family>
      <pointsize>72</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 72pt &quot;The Sunday&quot;;
    background-color: transparent;
    border: none;
    color: #FFFFFF; /* Text color */
}

QPushButton:hover {
    background-color: transparent;
    border: none;
    color: #CCCCCC; /* Optional: change text color slightly on hover */
}

QPushButton:pressed {
    background-color: transparent;
    border: none;
    color: #AAAAAA; /* Optional: darker text when pressed */
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_1_2">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>310</y>
      <width>181</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>The Sunday</family>
      <pointsize>72</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 72pt &quot;The Sunday&quot;;
    background-color: transparent;
    border: none;
    color: #FFFFFF; /* Text color */
}

QPushButton:hover {
    background-color: transparent;
    border: none;
    color: #CCCCCC; /* Optional: change text color slightly on hover */
}

QPushButton:pressed {
    background-color: transparent;
    border: none;
    color: #AAAAAA; /* Optional: darker text when pressed */
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>-50</y>
      <width>1101</width>
      <height>911</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/image/GameBoardPhoto.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>button_0_2</zorder>
   <zorder>button_0_1</zorder>
   <zorder>button_2_0</zorder>
   <zorder>scoreLabel</zorder>
   <zorder>ReturnButton</zorder>
   <zorder>button_1_0</zorder>
   <zorder>button_2_2</zorder>
   <zorder>button_0_0</zorder>
   <zorder>resetButton</zorder>
   <zorder>statusLabel</zorder>
   <zorder>button_1_1</zorder>
   <zorder>button_2_1</zorder>
   <zorder>button_1_2</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
